src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  /* Скидання параметрів браузера */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the '*' selector as it impacts performance. Set styles only for elements that require them.